README.md

```diff
@@ -1,5 +1,10 @@
 # Paragon
 
+## Warning: Package Moved
+
+This package will no longer receive updates and the latest version can be found
```

Comment:
[suggestion] At first glance, the way I read this was that Paragon itself will no longer be receiving updates, not just that older `@edx/paragon` NPM package won't be receiving further updates. 

Perhaps making "This package" more explicit by mentioning the old package name `@edx/paragon` instead as well as explicitly mentioning `@openedx/paragon` instead of just the URL to the package (i.e., non-zero effort to parse the package name from the URL):

> While the Paragon design system will continue to receive updates, these updates will no longer be published at the `@edx/paragon` NPM package. Instead, the latest versions can be found with [`@openedx/paragon`](https://www.npmjs.com/package/@openedx/paragon) instead.

Reply:
I like that suggested rewording!

Reply:
Yea, me too, I'll update it!